uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/modals/leave-request.js

```diff
@@ -0,0 +1,52 @@
+var modal = require('./modal');
+
+module.exports = (function () {
+  return modal.extend({
+
+    /**
+     * Selects tabs like comments or attachments
+     * @return {Object} this object
+     */
+    selectTab: function (tabName) {
+      var casper = this.casper;
+
+      casper.then(function () {
+        casper.click('div.chr_leave-request-modal__tab li[heading=' + tabName + '] a');
+      });
+
+      return this;
+    },
+
+    /**
+     * When user adds comments to the edit box, update the angular model in scope object
+     * @param {String} comment
+     * @return {Object} this object
+     */
+    addCommentToScope: function (comment) {
```

Comment:
its not adding to textbox, that would be a different call as explained in the call